doc/source/cluster/kubernetes.rst

```diff
@@ -1,271 +1,464 @@
+
+-  The `Operator <https://github.com/ray-project/ray/blob/master/python/ray/ray_operator/operator.py>`__ centralizes cluster launching and autoscaling in the `Operator pod <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/operator_configs/operator.yaml>`__. \
+   The user creates a `Kubernetes Custom Resource`_ describing the intended state of the Ray cluster. \
+   The Operator then detects the resource, launches a Ray cluster, and runs the autoscaling process in the operator pod. \
+   The Operator can manage multiple Ray clusters by running an autoscaling process for each Ray cluster.
+
+Comparision of configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The configuration options for the two methods are completely analogous - compare sample configurations for the `Cluster Launcher <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/kubernetes/example-full.yaml>`__
+and for the `Operator <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/operator_configs/example_cluster.yaml>`__.
+With a few exceptions, the fields of the RayCluster resource managed by the Operator are camelCase versions of the corresponding snake_case Cluster Launcher fields.
+In fact, the Operator `internally <https://github.com/ray-project/ray/blob/master/python/ray/ray_operator/operator_utils.py>`__ converts
+RayCluster resources to Cluster Launching configs.
+
+The details of these configuration differences are not essential to getting started, but here's a summary:
+
+- The Cluster Launching field ``available_node_types`` for specifiying the types of pods available for autoscaling is renamed to ``podTypes`` in the Operator's RayCluster configuration.
+- The Cluster Launching field ``resources`` for specifying custom Ray resources provided by a node type is renamed to ``rayResources`` in the Operator's RayCluster configuration.
+- The ``provider`` field in the Cluster Launching config has no analogue in the Operator's RayCluster configuration. (The Operator fills this field internally.)
+-  * When using the Cluster Launcher, ``head_ray_start_commands`` should include the argument ``--autoscaling-config=~/ray_bootstrap_config.yaml``; this is important for the configuration of the head node's autoscaler.
+   * On the other hand, the Operator's ``headRayStartCommands`` should include a ``--no-monitor`` flag to prevent the autoscaling/monitoring process from running on the head node.
+
+
+Quick Guide
+-----------
+
+The next two sections detail how to launch with the :ref:`Ray Cluster Launcher <k8s-cluster-launcher>`
+and with the :ref:`Ray Kubernetes Operator <k8s-operator>`, respectively.
+
+The section after that describes how to :ref:`interact <ray-k8s-interact>` with your Ray cluster
```

Comment:
Can you move this quick guide to the top (underneath "Ray supports ... bullet points")? This is basically a table of contents.

A list format would also be nice here (since it's currently just oddly spaced)